handlers.go

```diff
@@ -317,7 +318,23 @@ func (s *Server) ServeAPI() error {
 	if !s.cfg.DisableSwaggerEndpoint {
 		e.GET("/swagger/*", echoSwagger.WrapHandler)
 	}
-	return e.Start(s.cfg.ApiListen)
+
+	addRoutesWithNoPrefix(myecho)
+
+	return myecho.Start(s.cfg.ApiListen)
+}
+
+func addRoutesWithNoPrefix(myecho *echo.Echo) {
+	for _, value := range myecho.Routes() {
+		var newpath string
+		if strings.HasPrefix(value.Path, "/v1/") {
```

Comment:
if we introduce `v2`, this means we would need to add a condition here right? I suggest we add the following on the docs: 

1 - adding a new version programmatically. Add an instruction that a new version requires a new group example `v2`.
2 - how to separate the new version routes to a new file instead of putting them all in one handler.go.

Reply:
I think when we do V2 we'll want v2 to be in it's own separate file. I thought that was what you had suggested. So when we do v2 this whole handlers.go will be the v1 handler and we'll delete the addRoutesWithNoPrefix function and it's call and the v1 or v2 prefix will be required. Does that sound right?